Makefile

```diff
@@ -148,8 +152,8 @@ krew-plugin-manifest: yq bin/kubectl-volsync ## Build & package the kubectl plug
 	rm -f bin/kubectl-volsync.tar.gz
 	tar czf bin/kubectl-volsync.tar.gz LICENSE -C bin kubectl-volsync
 	HASH="`sha256sum bin/kubectl-volsync.tar.gz | cut -f 1 -d ' '`" \
-	VERSION="$(VERSION)" \
-	$(YQ) --inplace '.spec.version=strenv(VERSION) | with(.spec.platforms[]; .sha256=strenv(HASH) | .uri|=sub("v[[:digit:]]+\.[^/]+", strenv(VERSION)))' ./kubectl-volsync/volsync.yaml
+	BUILD_VERSION="$(BUILD_VERSION)" \
+	$(YQ) --inplace '.spec.version=strenv(BUILD_VERSION) | with(.spec.platforms[]; .sha256=strenv(HASH) | .uri|=sub("v[[:digit:]]+\.[^/]+", strenv(BUILD_VERSION)))' ./kubectl-volsync/volsync.yaml
```

Comment:
These need to stay as VERSION since they are used to set the url where the CLI can be retrieved from. The path will be determined by the release tag (i.e., v0.4.0)

Reply:
@JohnStrunk I pushed a change, which I think is correct - can you take a look?  I basically did this:

```
VERSION="v$(VERSION)" \
```

Note the `v`.  Since $(VERSION) will simply be `0.4.0`